sources/Interop/D3D12MemoryAllocator/src/ALLOCATION_FLAGS.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+namespace TerraFX.Interop
+{
+    /// <summary>Bit flags to be used with ALLOCATION_DESC::Flags.</summary>
+    public enum ALLOCATION_FLAGS
```

Comment:
```suggestion
    [Flags]
    public enum ALLOCATION_FLAGS
```

requires adding `using System;` as well

Reply:
Added in ce4f364c5c44a49f25bc206df77c142809a7bcc7 🙂